LoopKit/InsulinKit/HKQuantitySample+InsulinKit.swift

```diff
@@ -40,17 +40,13 @@ extension HKQuantitySample {
         var metadata: [String: Any] = [
             HKMetadataKeySyncVersion: syncVersion,
             HKMetadataKeySyncIdentifier: syncIdentifier,
+            HKMetadataKeyWasUserEntered: dose.manuallyEntered,
             MetadataKeyHasLoopKitOrigin: true,
             MetadataKeyManuallyEntered: dose.manuallyEntered
         ]
         
         switch dose.type {
         case .basal, .tempBasal, .suspend:
-            // Ignore 0-duration basal entries
-            guard dose.endDate.timeIntervalSince(dose.startDate) > .ulpOfOne else {
```

Comment:
Lantus (basal) is injected at a single time. Technically does not have an end date. I can not find guidance how to log injected basal insulin. For instance, if I take 14units, do you set the start/end date to the same? Or set the end date to 24 hours in the future? 🤷  So because of that removing this guard.